payments_service/braintree/views.py

```diff
@@ -130,7 +130,3 @@ def perform_content_negotiation(self, request, **kw):
             return (renderer, renderer.media_type)
         return (super(self.__class__, self)
                 .perform_content_negotiation(request, **kw))
-
-    def get(self, request, **kw):
-        return super(self.__class__, self).get(
-            request, **dict(request.QUERY_PARAMS.items()))
```

Comment:
oh, heh, I missed this on first review. Yeah, it was just an oversight in the bodyguard. The bodyguard passes ``**request.REQUEST`` though -- is that ok?